src/generators/model-template.handlebars

```diff
@@ -1,23 +1,31 @@
 import { BaseTable, Column } from '@outerbase/query-builder';
 {{#each references}}
-import { {{ capitalize (camelCase table) }} } from './{{#if (neq schema "public")}}{{schema}}/{{/if}}{{ camelCase table }}';
+    import {
+    {{capitalize (camelCase table)}}
+    } from './{{#if (neq schema 'public')}}{{schema}}/{{/if}}{{camelCase
+        table
+    }}';
```

Comment:
I don't think I'm "thrilled" with how this would look in the generated model code, I'd prefer it was all on one line. I'm sure this is part of the prettier changes being applied everywhere but I would want this file to be an exception to the rules being applied globally.